com.unity.shadergraph/Tests/Generation/GenerationTests.cs

```diff
@@ -48,20 +45,13 @@ private static Texture2D DrawToTex(Shader shader)
             var prevActive = RenderTexture.active;
             RenderTexture.active = rt;
             Graphics.Blit(null, rt, new Material(shader));
-            Texture2D output = new Texture2D(4, 4, TextureFormat.ARGB32, false);
+            Texture2D output = new(4, 4, TextureFormat.ARGB32, false);
             output.ReadPixels(new Rect(0, 0, 4, 4), 0, 0);
             RenderTexture.active = prevActive;
             rt.Release();
             return output;
         }
 
-        static object[] testAsIsSource = new object[]
```

Comment:
Why was this removed?

Reply:
The test case source still refers to it

Reply:
If the intent was removing tests, that whole test needs to be removed alongside

Reply:
I missed the reference in to this var in the attribute. Thanks for pointing it out.

Reply:
Updated.